devel/fortio/README.md

```diff
@@ -6,10 +6,13 @@ The name fortio comes from greek φορτίο which is load/burden.
 
 ## Command line arguments
 
+Fortio can be and http or grpc load generator, gathering statistics using the `load` command, or start simple http and grpc ping servers with the `server` command or issue grpc ping messages using the `grpcping` command.
+
 ```
 $ fortio
-Φορτίο 0.2.0 usage:
-	fortio [flags] target
+Φορτίο 0.2.2 usage:
```

Comment:
I don't really care, I just don't think it provides value at the expense of being broken for some terminals. Doesn't particularly matter to me.